wordstats/file_handling/cognate_files_path.py

```diff
@@ -1,39 +1,33 @@
-
-from .config import DATA_FOLDER_COGNATES, WHITELIST, BLACKLIST, RULES, CANDIDATES
+from config import DATA_FOLDER_COGNATES, WHITELIST, BLACKLIST, RULES, CANDIDATES
 import os
-import codecs
 
-# useful methods for handling files
 
 def path_of_cognate_languages(primary, secundary):
     package_directory = os.path.dirname(os.path.abspath(__file__))
     file_path = package_directory + os.sep + "{0}{1}{2}{3}".format(DATA_FOLDER_COGNATES, os.sep, primary, secundary)
     return file_path
 
-def path_of_cognate_candidates(primary, secundary, method_name):
 
+def path_of_cognate_candidates(primary, secundary, method_name):
     return path_to_cognate_file(primary, secundary, CANDIDATES, method_name)
 
 
-def path_of_cognate_blacklist(primary, secundary, author:str = ""):
```

Comment:
note that many of these changes are also PyCharm reformatting code after a refactoring.